src/components/userauth/ErrorMessage/index.test.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import ErrorMessage from ".";
+
```

Comment:
"it should return null if there is no error"

Reply:
In this case returning null doesn't mean that something is wrong. Message only displayed when user makes mistake. Most of the time there shouldn't be a message.

Reply:
I feel like im missing your point.

Reply:
your code is correct but i'm aware that returning null doesnt mean something is wrong. its intended, thats why we write a test case to express that intention

Reply:
but in general if you have only 1 test and it all it returns is null its useless for a snapshot. so in this case we definitely want the non-null case covered.

Reply:
for the null case we wouldnt normally use a snapshot test to be honest. we'd just use jest expect(foo).toBeNull

Reply:
I'm also not sure why exactly it is, but null type is defined as object ( I've just found it via MDN [link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/null) and in the code, message is defined as string. This difference may cause a problem.

Reply:
hmm I see. Thank you Louis